### DISCUSSION TITLE
Convert bdd feature files into gauge format.

### DISCUSSION THREAD

**Discussion by sanjeeth-07** [getgauge/gauge#2024] [General]
Is there any plugin available to convert normal BDD format feature files into gauge specific format.

**Comment by zabil** [comment]
There's no official plugin for BDD conversion

There are a few tips about migrating from cucumber here
https://gauge.org/2019/07/22/the-gauge-move/

There's also a third party tool to convert gherkin to markdown.
https://github.com/raviqqe/gherkin2markdown

You could modify the converted markdown to make it easier to write Gauge specifications.

**Reply by sanjeeth-07** [reply]
Thanks for sending me the links @zabil
